scripts/verify-template.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const puppeteer = require('puppeteer');
```

Comment:
Nice! This would have been a perfect opportunity to play around with [`playwright`](https://github.com/microsoft/playwright) as well, but puppeteer is great 👍

Reply:
I figured `puppeteer` would be simpler because it’s just one browser whereas `playwright` is multiple which might have more complicated native dependency requirements that aren’t met by the GitHub Actions base image.